<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>776</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>320</y>
      <width>190</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    /* Основной стиль кнопки */
    appearance: none;
    border: 0;
	
	/*background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0.0610329 rgba(4,83,125, 255), stop:0.629108 rgba(14,120,125, 255));*/
	background-color: rgb(4, 83, 125);
    color: #fff;
    padding: 8px 16px;
    font-size: 16px;
    box-shadow: 0 0 0 4px #cbd6ee;
}

QPushButton:pressed {
    /* Стиль кнопки при нажатии */
     /* Например, изменяем цвет фона */
	/*background-color: rgb(121,119,143);*/
	
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.653, y2:0.596591, stop:0 rgba(4, 83, 125, 255), stop:0.99061 rgba(121, 119, 143, 255));
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(4, 83, 125, 255), stop:1 rgba(121, 119, 143, 255));
    box-shadow: inset 0 0 0 4px #cbd6ee; /* Добавляем эффект &quot;продавливания&quot; */
}</string>
    </property>
    <property name="text">
     <string>Авторизироваться</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>192</x>
      <y>184</y>
      <width>392</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> border: 0;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>192</x>
      <y>230</y>
      <width>392</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> border: 0;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_1">
    <property name="geometry">
     <rect>
      <x>192</x>
      <y>246</y>
      <width>401</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>_________________________________________________________________</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>192</x>
      <y>200</y>
      <width>401</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>_________________________________________________________________</string>
    </property>
   </widget>
   <widget class="QLabel" name="image">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>50</y>
      <width>128</width>
      <height>128</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>img/user.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>118</x>
      <y>40</y>
      <width>540</width>
      <height>371</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-30</y>
      <width>792</width>
      <height>321</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(4, 83, 125);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>250</y>
      <width>781</width>
      <height>281</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(121, 119, 143);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>192</x>
      <y>270</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Запомнить меня</string>
    </property>
   </widget>
   <zorder>label_4</zorder>
   <zorder>label_3</zorder>
   <zorder>label</zorder>
   <zorder>pushButton</zorder>
   <zorder>lineEdit</zorder>
   <zorder>lineEdit_2</zorder>
   <zorder>label_1</zorder>
   <zorder>label_2</zorder>
   <zorder>image</zorder>
   <zorder>checkBox</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
